### PR TITLE
[powermax] Disable RFC2217 in favor of IP conection thing type

### LINKED ISSUE #4849: [powermax] Compatibility with RFC2217 ?
I plan to update the Powermax binding to make it fully compatible with serial over IP.
It should be easy.

### PR DESCRIPTION
**Pull request by lolodomo**
Fix #4849 

Signed-off-by: Laurent Garnier <lg.hc@free.fr>

### CODE REVIEW

**Comment by lolodomo** [comment]
Fix #4849

**Comment by lolodomo** [comment]
Any chance to get a quick code review ? This is only 5 new lines of code + few lines added in the README.

File: addons/binding/org.openhab.binding.powermax/README.md [hunk 1]
```diff
@@ -2,7 +2,15 @@
 
 Visonic produces the Powermax alarm panel series (PowerMax, PowerMax+, PowerMaxExpress, PowerMaxPro and PowerMaxComplete) and the Powermaster alarm series (PowerMaster 10 and PowerMaster 30). This binding allows you to control the alarm panel (arm/disarm) and allows you to use the Visonic sensors (movement, door contact, ...) within openHAB.
 
-The PowerMax provides support for a serial interface that can be connected to the machine running openHAB. The serial interface is not installed by default but can be ordered from any PowerMax vendor (called the Visonic RS-232 Adaptor Kit).
+The PowerMax provides support for a serial interface. This serial interface is not installed by default but can be ordered from any PowerMax vendor (called the Visonic RS-232 Adaptor Kit).
```
Comment (cweitkamp) [inline addons/binding/org.openhab.binding.powermax/README.md]: One minor remark for future PRs. Please put new sentences in a separate row.

```suggestion
The PowerMax provides support for a serial interface.
This serial interface is not installed by default but can be ordered from any PowerMax vendor (called the Visonic RS-232 Adaptor Kit).
```

File: addons/binding/org.openhab.binding.powermax/README.md [hunk 2]
```diff
@@ -2,7 +2,15 @@
...
+
+In case your alarm panel is directly connected to a serial port of your openHAB server (or to an USB port through a serial to USB converter), you must setup a serial connection thing type in openHAB.
+
+But you even don't need to have your alarm panel directly connected to your openHAB server. You can connect it for example to a Raspberry PI and use [ser2net Linux tool](https://sourceforge.net/projects/ser2net/) to make the serial connection available on LAN (serial over IP). In this case, you must setup an IP connection thing type in openHAB. Here is an example of ser2net.conf you can use to share your serial port /dev/ttyUSB0 on IP port 4444:
```
Comment (cweitkamp) [inline addons/binding/org.openhab.binding.powermax/README.md]:
```suggestion
But you even don't need to have your alarm panel directly connected to your openHAB server.
You can connect it for example to a Raspberry PI and use [ser2net Linux tool](https://sourceforge.net/projects/ser2net/) to make the serial connection available on LAN (serial over IP).
In this case, you must setup an IP connection thing type in openHAB.
Here is an example of ser2net.conf you can use to share your serial port /dev/ttyUSB0 on IP port 4444:
```

**Pull request merged**